test/serialize.js

```diff
@@ -61,6 +131,14 @@ test('parse->serialize', function() {
       cookie.serialize('cat', ' ";/')));
 });
 
+test('encoded method validation', function() {
+   assert.throws(cookie.serialize.bind(null, 'cat', ' ', {
+       encode: 42
+   }), /number is not a function/);
```

Comment:
Looks like this is failing on io.js 2.x due to a difference in the v8 error message.